### PR TITLE
Update minor tasklist changes

### PR DESCRIPTION
**Pull request by louwie17**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Two items:
- Fix the progressive header and show `Welcome to <site name>` when all tasks are complete (see second to last point in the original issue: https://github.com/woocommerce/woocommerce/issues/32163)
- Add loading placeholder for sectioned task list

### How to test the changes in this Pull Request:

1. Install and enable the [WCA test helper](https://github.com/woocommerce/woocommerce-admin-test-helper/)
2. Navigate to Tools->WCA Test Helper->Features
3. Toggle the `tasklist-setup-experiment-2` experiment to `true`
5. Navigate to the homescreen and notice the sectioned task list loading placeholder (it looks different from the original)
6. Finish all the tasks
7. Once finished notice how the progressive header is still present but without the progress bar and sub text. Just the **Welcome to <site name>**

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?
* [ ] Have you created a changelog file by running `pnpm nx affected --target=changelog`?

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list-placeholder.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	numTasks?: number;
+	query: {
+		task?: string;
+	};
+};
+
+const SectionedTaskListPlaceholder: React.FC< TasksPlaceholderProps > = (
+	props
+) => {
+	const { numTasks = 3 } = props;
```
Comment (joshuatf) [inline plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list-placeholder.tsx:16]: Nice 👍

**Review by joshuatf — DISMISSED** [review]
Testing well and LGTM! 🚢

File: plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list-placeholder.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
```
Comment (octaedro) [inline plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list-placeholder.tsx]: It looks like this is not being used.

**Review by joshuatf — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**